composer.json

```diff
@@ -13,14 +15,14 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^7.1"
     },
     "minimum-stability": "dev",
     "prefer-stable": true,
     "extra": {
         "branch-alias": {
-            "dev-master": "2.6-dev",
-            "dev-develop": "2.7-dev"
+            "dev-master": "3.0-dev",
+            "dev-develop": "3.1-dev"
```

Comment:
Don't change these until the develop branch is merged to master (i.e., right before the 3.0 release).

When you do, use the format `3.0.x-dev` and `3.1.x-dev` (this is the recommended format by Composer).